catalyst-gateway-crates/c509-certificate/examples/web/index.js

```diff
@@ -0,0 +1,68 @@
+// Testing the wasm binding JS functions.
+
+import init, {
+	generate,
+	verify,
+	decode,
+	PublicKey,
+	PrivateKey,
+} from "../../pkg/c509_certificate.js";
+
+const pem_sk = `
+-----BEGIN PRIVATE KEY-----
+MC4CAQAwBQYDK2VwBCIEIP1iI3LF7h89yY6QZmhDp4Y5FmTQ4oasbz2lEiaqqTzV
+-----END PRIVATE KEY-----
+`;
+
+const pem_pk = `
+-----BEGIN PUBLIC KEY-----
+MCowBQYDK2VwAyEAtFuCleJwHS28jUCT+ulLl5c1+MXhehhDz2SimOhmWaI=
+-----END PUBLIC KEY-----
+`;
+
+const tbs = {
+    c509_certificate_type: 0,
+    certificate_serial_number: 1000000n,
+    issuer: {
+        RelativeDistinguishedName: [
+            {
+                oid: "2.5.4.3",
+                value: [{ Text: "RFC test CA" }],
+            },
+        ],
+    },
+    validity_not_before: 1_672_531_200n,
+    validity_not_after: 1_767_225_600n,
+    subject: { Text: "01-23-45-ff-fe-67-89-AB" },
+    subject_public_key_algorithm: {
+        oid: "1.3.101.112",
+    },
+    subject_public_key: [],
+    extensions: [
+        {
+            oid: "2.5.29.19",
+            value: { Int: -2n },
+            critical: false,
+        },
+    ],
+    issuer_signature_algorithm: {
+        oid: "1.3.101.112",
+    },
+};
+
+async function run() {
+	await init();
```

Comment:
something is probably wrong with the formatter, the indents are random

Reply:
The indent looks fine in my local, will try to fix the format again

Reply:
Ok, then I think locally you might be using tabs and here are spaces or vice-versa ;P quite common issue, make sure everywhere it's the same